src/Apps/Order/Routes/__tests__/Status.test.tsx

```diff
@@ -243,6 +246,9 @@ describe("Status", () => {
           state: "SUBMITTED",
         })
         expect(page.text()).toContain("Your order has been submitted")
+        expect(page.text()).toContain(
+          "You will receive a confirmation email by January 15"
```

Comment:
Think these should be `Jan 15` not `January 15`

Reply:
yes, since we are mocking MP response, this is whatever that was mocked, but let me update to be consistent with what we actually will see 👍